test/cypress/support/commands.js

```diff
@@ -347,7 +354,14 @@ Cypress.Commands.add('giCreateGroup', (name, {
       }).then(() => {
         const router = sbp('controller/router')
         if (router.history.current.path === '/dashboard') return
-        return router.push({ path: '/dashboard' })
+        const state = sbp('state/vuex/state')
+        return sbp('gi.actions/identity/setGroupAttributes', {
+          contractID: state.loggedIn.identityContractID,
+          data: {
+            groupContractID: state.currentGroupId,
+            attributes: { seenWelcomeScreen: true }
+          }
+        }).then(() => router.push({ path: '/dashboard' }))
```

Comment:
Is there a way this can be DRY'd? (Repeated above)

Reply:
I'm not entirely sure. It could maybe be a Cypress command.